pyproject.toml

```diff
@@ -26,7 +26,8 @@ dependencies = [
     # pandas: "excel" is an extra that contains xlrd that is necessary for reading old .xls Excel files
     "pyld>=2.0.4",
     "pyyaml>=6.0.2",
-    "rdflib>=7.0.0",
+    "rdflib==7.0.0",
+    # rdflib: don't upgrade until ...
```

Comment:
@Nora-Olivia-Ammann I think it's better to merge this PR now, and add a note here. Can you complete this comment?